internal/action/sync.go

```diff
@@ -32,6 +33,8 @@ func init() {
 		return
 	}
 
+	debug.Log("GOPASS_AUTOSYNC_INTERVAL is deprecated. Please use autosync.interval")
```

Comment:
I guess this is a silent `debug.Log` instead of a noisy `out.Warning` because of the lack of context in `init`?